editor/SCsub

```diff
@@ -97,7 +97,10 @@ void register_exporter_types() {{
     # Generated with `make include-list` for each resource.
 
     # Editor translations
-    tlist = glob.glob(env.Dir("#editor/translations/editor").abspath + "/*.po")
+    if env["editor_localization"]:
+        tlist = glob.glob(env.Dir("#editor/translations/editor").abspath + "/*.po")
```

Comment:
According to [a recent rewrite](https://github.com/godotengine/godot/pull/98653/files#diff-5d2591fb5837decc3fec5e4992821a185ad895888924e4e736a31c7756c067f8R99) of old glob to a newer use just: `tlist = Glob("#editor/translations/editor/*.po")`